benchmarks/run_benchmark.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -eo pipefail
+set +x
+set -u
+
+function cleanup() {
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        docker rm -f $(docker ps -a -q) || true
```

Comment:
This seems dangerous; what if other Docker images are running?

Reply:
Removed this. The OSX variant will need a little bit of love anyway it broke somewhere along the way. I can do that in another iteration - after we land this PR - if someone needs that.